src/bot.py

```diff
@@ -79,7 +80,12 @@
 elif use_file_variables == True:
     logging.info("Using file variables.")
     try:
-        from credentials import *
+        config = configparser.ConfigParser()
+        config.read('config.txt')
+       	CONSUMER_KEY = config['credentials']['CONSUMER_KEY']

```

Comment:
There's an indentation error over here.

Reply:
I am submitting a properly formatted file, I am not getting why it's getting an indentation error after uploading.

> There's an indentation error over here.

I am submitting a properly formatted file, I am not getting why it's getting an indentation error after uploading.

Reply:
I am submitting a properly formatted file, I am not getting why it's getting an indentation error after uploading.

Reply:
@sopneshgandhi Maybe you could try editing it with GitHub?